format/spec.md

```diff
@@ -889,6 +890,7 @@ Table metadata consists of the following fields:
 | _optional_ | _optional_ | _optional_ | **`partition-statistics`**  | A list (optional) of [partition statistics](#partition-statistics).                                                                                                                                                                                                                                                                                                                              |
 |            |            | _optional_ | **`row-lineage`**           | A boolean, defaulting to false, setting whether or not to track the creation and updates to rows in the table. See [Row Lineage](#row-lineage).                                                                                                                                                                                                                                                  |
 |            |            | _optional_ | **`next-row-id`**           | A `long` higher than all assigned row IDs; the next snapshot's `first-row-id`. See [Row Lineage](#row-lineage).                                                                                                                                                                                                                                                                                  |
+|            |            | _optional_ | **`current-key-id`**        | ID of the encryption key that encrypts the manifest list key metadata.                                                                                                                                                                                                                                                                                  |
```

Comment:
I'm not sure I agree with using "current" here. I don't see how this key ID would be "current" -- it is either the key for the manifest list or it is not. How about using just `key-id`?

Reply:
sounds good, changing to `key-id`

Reply:
Sorry, my comment was based on thinking this was a snapshot's key ID because of the description. This was actually in table metadata.

Can we trust table metadata to provide a key ID to use? I would probably have the encryption manager select the key automatically -- the one that is available from KMS.

Reply:
Agreed, we can have this handled by the encryption manager.